src/coreclr/runtime-prereqs.proj

```diff
@@ -0,0 +1,27 @@
+<Project Sdk="Microsoft.Build.NoTargets">
+  <PropertyGroup>
+    <NativeVersionFile>$(ArtifactsObjDir)/coreclr/_version.h</NativeVersionFile>
+    <RuntimeVersionFile>$(ArtifactsObjDir)/coreclr/runtime_version.h</RuntimeVersionFile>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include=".nuget/optdata/optdata.proj" />
+    <ProjectReference Condition="'$(TargetOS)' == 'Windows'" Include="ToolBox/SOS/DacTableGen/DacTableGen.csproj" />
+  </ItemGroup>
+
+  <Import Project="$(RepositoryEngineeringDir)versioning.targets" />
+
+  <Target Name="OutputPgoPathForCI" Condition="'$(ContinuousIntegrationBuild)' == 'true'">
```

Comment:
Why not move that target into optdata.proj (and sequence it as an aftertargets of `GetPgoDataPackagePath` to avoid the MSBuild task here?